config/crd/kustomization.yaml

```diff
@@ -8,12 +8,12 @@ resources:
 patchesStrategicMerge:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
 # patches here are for enabling the conversion webhook for each CRD
-#- patches/webhook_in_servicebindings.yaml
+# - patches/webhook_in_servicebindings.yaml
```

Comment:
look like no-op change.

Reply:
@pedjak It seems this change comes from ea79796cba3f5a82e2343d760785179eaa02c3fe, so I'd leave it be unless there's a stronger reason to revert this.